enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -234,4 +235,26 @@ describe("InteractiveQuestion", () => {
       screen.queryByTestId("chart-type-selector-button"),
     ).not.toBeInTheDocument();
   });
+
+  // Obviously, we can't test every single permutation of chart settings right now, but tests in the core
+  // app should cover most cases anyway.
+  it("should allow user to use chart settings", async () => {
+    setup({ withChartTypeSelector: true });
+    await waitForLoaderToBeRemoved();
+
+    await userEvent.click(screen.getByLabelText("gear icon"));
+
+    const popover = within(screen.getByRole("dialog"));
+    expect(popover.getByTestId("chartsettings-sidebar")).toBeInTheDocument();
+    await userEvent.click(screen.getByTestId("Test Column-settings-button"));
+
+    const columnTitle = screen.getByTestId("column_title");
+    await userEvent.clear(columnTitle);
+    await userEvent.type(columnTitle, "A New Test Column");
+    await userEvent.tab();
+
+    expect(
+      await screen.findByTestId("draggable-item-A New Test Column"),
```

Comment:
I wonder if it'd make more sense to assert the actual text? It seems a bit weird to have to resort to checking the test ID for this.